Card/src/main/java/com/paypal/android/card/CardClient.kt

```diff
@@ -18,4 +22,26 @@ class CardClient internal constructor(private val cardAPI: CardAPI) {
      */
     suspend fun approveOrder(request: CardRequest): CardResult =
         cardAPI.confirmPaymentSource(request.orderID, request.card)
+
+    /**
+     * Confirm [Card] payment source for an order. Use this method for Java integrations
+     *
+     * @param request [CardRequest] for requesting an order approval
+     * @param callback callback to get response
+     */
+    fun approveOrder(request: CardRequest, callback: ApproveOrderCallback) {
+        CoroutineScope(Dispatchers.IO).launch {
+            try {
+                val result = approveOrder(request)
+                callback.success(result)
+            } catch (e: PayPalSDKError) {
+                callback.failure(e)
+            }
+        }
+    }
+}
+
+interface ApproveOrderCallback {
```

Comment:
Maybe this should live in its own file and have doc strings? Not sure what convention we want to follow for callbacks in this project.